alvr/server/src/connection.rs

```diff
@@ -642,10 +643,38 @@ fn try_connect(mut client_ips: HashMap<IpAddr, String>) -> ConResult {
         thread::spawn(|| ())
     };
 
-    let tracking_manager = Arc::new(Mutex::new(TrackingManager::new()));
+    let tracking_manager: Arc<
+        alvr_common::parking_lot::lock_api::Mutex<
+            alvr_common::parking_lot::RawMutex,
+            TrackingManager,
+        >,
+    > = Arc::new(Mutex::new(TrackingManager::new()));
+    let hand_gesture_manager: Arc<
+        alvr_common::parking_lot::lock_api::Mutex<
+            alvr_common::parking_lot::RawMutex,
+            HandGestureManager,
+        >,
+    > = Arc::new(Mutex::new(HandGestureManager::new()));
```

Comment:
I think you can omit the variable type. In case you really need to specify the type, don't use fully qualified paths, but import the types you need at the top of the file.